### PR TITLE
Suppress NoSuchKey error in REST API

### LINKED ISSUE #8055: NoSuchKey stacktrace in REST API
### Description

Getting high log error rate alerts for `NoSuchKey` in REST API.

### Steps to reproduce

Query state proof API

### Additional context

```
2024-04-09T02:17:22.585Z ERROR c89c889a Failed to download {"Bucket":"hedera-mainnet-streams","Key":"recordstreams/record0.0.31/2024-04-09T02_17_10.008160293Z.rcd_sig","RequestPayer":"requester"} NoSuchKey: The specified key does not exist.
    at de_NoSuchKeyRes (/home/node/app/node_modules/@aws-sdk/client-s3/dist-cjs/index.js:4809:21)
    at de_CommandError (/home/node/app/node_modules/@aws-sdk/client-s3/dist-cjs/index.js:4747:19)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /home/node/app/node_modules/@smithy/middleware-serde/dist-cjs/index.js:35:20
    at async /home/node/app/node_modules/@aws-sdk/middleware-signing/dist-cjs/index.js:225:18
    at async /home/node/app/node_modules/@smithy/middleware-retry/dist-cjs/index.js:320:38
    at async /home/node/app/node_modules/@aws-sdk/middleware-flexible-checksums/dist-cjs/index.js:173:18
    at async /home/node/app/node_modules/@aws-sdk/middleware-sdk-s3/dist-cjs/index.js:97:20
    at async /home/node/app/node_modules/@aws-sdk/middleware-sdk-s3/dist-cjs/index.js:120:14
    at async /home/node/app/node_modules/@aws-sdk/middleware-logger/dist-cjs/index.js:33:22 {
  '$fault': 'client',
  '$metadata': {
    httpStatusCode: 404,
    requestId: undefined,
    extendedRequestId: undefined,
    cfId: undefined,
    attempts: 1,
    totalRetryDelay: 0
  },
  Code: 'NoSuchKey',
  Details: 'No such object: hedera-mainnet-streams/recordstreams/record0.0.31/2024-04-09T02_17_10.008160293Z.rcd_sig'
}
```

### Hedera network

mainnet

### Version

v0.101.0

### Operating system

None

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Description**:

* Add a mirror node specific contributing guide
* Fix `bootBuildImage` Gradle task on arm64
* Fix acceptance tests startup probe sometimes failing after importer catches up
* Remove reactor dependency from common module
* Suppress `NoSuchKey` error in REST API

**Related issue(s)**:

Fixes #8055

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,32 +51,13 @@ expose the data. This approach provides the following advantages:
...
-a [balance file](https://github.com/hashgraph/hedera-protobufs/blob/main/streams/account_balance_file.proto). The
-balance file contains the list of Hedera accounts and their corresponding HBAR and token balances. This file is
-processed by the mirror node in a similar fashion as outlined above.
-
 ## Getting Started
 
-### Technologies
-
-Multiple technologies are utilized in the mirror node. The following topics are areas where basic knowledge is valuable
-to understanding the mirror node operations.
```
Comment (steven-sheehy) [inline README.md:64]: Removing this awkwardly worded section as it's not appropriate for a top-level README to list random technologies.

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Labels are used to group related issues together and for various release process tasks. Only maintainers have permission
+to add or remove labels to issues. For other contributors, project maintainers will adjust the labels as necessary to
+meet the project's requirements. Every issue should have one of these mandatory `Type` labels:
+
+- `bug`
+- `dependencies`
+- `documentation`
+- `enhancement`
+
+This label will be used for grouping of items in the release notes. Additionally, issues are also recommended to have a
```
Comment (jascks) [inline docs/contributing.md:25]: Either `for the grouping of items` or `for grouping items`.

Comment (steven-sheehy) [reply docs/contributing.md]: Fixed

**Review by xin-hedera — APPROVED** [review]
LGTM

**Review by jascks — APPROVED** [review]
LGTM

**Pull request merged**